src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeedItem.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * @flow
+ * @file Active state component for Activity Feed
+ */
+import * as React from 'react';
+import classNames from 'classnames';
+import getProp from 'lodash/get';
+import { type IntlShape } from 'react-intl';
+import AppActivity from '../app-activity';
+import Comment from '../comment';
+import TaskNew from '../task-new';
+import Version, { CollapsedVersion } from '../version';
+import Keywords from '../keywords';
+import withErrorHandling from '../../withErrorHandling';
+import type { FocusableFeedItemType } from '../../../../common/types/feed';
+
+type Props = {
+    action: string | 'applied',
```

Comment:
This `action` prop is required here but I didn't see it in the `ActivityFeed` when you use `ActivityFeedItem`.